### PR TITLE
Jetpack Top Posts: improve styles and make image larger

### LINKED ISSUE #1830: Top Posts & Pages: increase image size and review styles
**Is your feature request related to a problem? Please describe.**
We have a few pubs using Jetpack's Top Posts & Pages widget/shortcode. It's still using the default image size (40px x 40px) but it would be nice to increase that to something that's easier to see/more comparable with how the Homepage Posts block can be styled.

The styles should also be reviewed for when this is added as a widget or shortcode, with and without AMP, to make sure things are styled consistently (in some cases the posts appear with their list markup). 

Related docs: https://jetpack.com/blog/custom-thumbnails-top-posts-widget/

### PR DESCRIPTION
**Pull request by laurelfulford**
### All Submissions:

* [x] Have you followed the [Newspack Contributing guideline](https://github.com/Automattic/newspack-theme/blob/master/.github/CONTRIBUTING.md)?
* [x] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) and [VIP Go coding standards](https://vip.wordpress.com/documentation/vip-go/code-review-blockers-warnings-notices/)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?

### Changes proposed in this Pull Request:

This PR makes some minor improvements to the Jetpack Top Posts & Pages widget, since we use it somewhat often on sites. It:

* Makes sure the widget picks up the right font styles
* Increases the size of the image in the list view
* Makes sure the widget is styled correctly when used in the content as a short-code. 

Closes #1830 

### How to test the changes in this Pull Request:

1. Start on a test site running Jetpack, and make sure it's connected to WP.com and that the Extra Sidebar Widgets are enabled under https://[yoururl.com]/wp-admin/admin.php?page=jetpack_modules (or by going WP Admin > Jetpack and clicking the Modules link at the bottom of the screen.
2. Create a new page and add the following code:

```
<!-- wp:columns -->
<div class="wp-block-columns"><!-- wp:column -->
<div class="wp-block-column"><!-- wp:shortcode -->
[jetpack_top_posts_widget title="Most Read" count="6" types="post" display="images"]
<!-- /wp:shortcode -->

<!-- wp:shortcode -->
[jetpack_top_posts_widget title="Most Read" count="6" types="post" display="list"]
<!-- /wp:shortcode -->

<!-- wp:shortcode -->
[jetpack_top_posts_widget title="Most Read" count="6" types="post" display="grid"]
<!-- /wp:shortcode --></div>
<!-- /wp:column -->

<!-- wp:column -->
<div class="wp-block-column"></div>
<!-- /wp:column --></div>
<!-- /wp:columns -->
```

3. Publish.
4. Navigate to the Customizer and add three Top Posts & Pages widgets to the sidebar: one set to list view, one with image list, and one set to grid view.
5. View on the front-end -- this is mostly for comparison's sake. The widgets should look mostly okay, though they may not be picking up your custom fonts, and the images in the list view are super small.

<details>
<summary><strong>Top Posts shortcode in content</strong></summary>

image

image

</details>

<details>
<summary><strong>Top Posts widget in sidebar</strong></summary>

image

image
</details>

6. Apply the PR and run `npm run build`.
7. Confirm that the widgets all look good; the list view should also have larger images. 

<details>
<summary><strong>Top Posts shortcode in content</strong></summary>

image

image

</details>

<details>
<summary><strong>Top Posts widget in sidebar</strong></summary>

image

image

</details>

8. Remove all the Top Posts widgets from the sidebar. 
9. Confirm that the shortcodes are still styled the same (if you were to switch to `master` now, they appear unstyled).  
11. On this branch, navigate to Customizer > Typography and change your Heading font. Confirm this font is used for the post titles in the shortcode and widget versions of the Top Posts.

### Other information:

* [x] Have you added an explanation of what your changes do and why you'd like us to include them?
* [ ] Have you written new tests for your changes, as applicable?
* [ ] Have you successfully ran tests with your changes locally?

### CODE REVIEW

File: newspack-theme/inc/typography.php [hunk 1]
```diff
@@ -142,6 +142,8 @@ function newspack_custom_typography_css() {
 		#jp-relatedposts.jp-relatedposts,
 		.jp-relatedposts-i2 .jp-relatedposts-headline,
 		#jp-relatedposts.jp-relatedposts .jp-relatedposts-headline,
+		.jetpack_top_posts_widget li a,
+		..widget_top-posts li a,
```
Comment (dkoo) [inline newspack-theme/inc/typography.php:146]: is the `..` a typo?

```suggestion
		.widget_top-posts li a,
```

**Review by dkoo — CHANGES_REQUESTED** [review]
Working well! Just saw one weird thing in the CSS.

File: newspack-theme/inc/typography.php (hunk 1 above)
Comment (laurelfulford) [reply newspack-theme/inc/typography.php]: 🤦‍♀️ Thanks Derrick! This should be fixed in [0e868f3](https://github.com/Automattic/newspack-theme/pull/2046/commits/0e868f3a5acb106b857802e2bb5f8d54cdee20d2)!

Comment (dkoo) [reply newspack-theme/inc/typography.php]: Confirmed that the fix corrected some strange typography styling issues I was seeing!

**Review by dkoo — APPROVED** [review]

File: newspack-theme/inc/typography.php (hunk 1 above)
Comment (laurelfulford) [reply newspack-theme/inc/typography.php]: Always nice when a single character typo takes out a whole stylesheet 😆

**Pull request merged**